### PR TITLE
Create file bundle schema

### PR DESCRIPTION
**Pull request by skoehnen**
Implementation of #80

### CODE REVIEW

File: v3.0/products/fileBundle.schema.json [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+      "type": "string",
+      "description": "Metadata node type (schema).",
+      "const": "https://openminds.ebrains.eu/core/fileBundle"
+    },
+    "@id": {
+      "type": "string",
+      "description": "Metadata node identifier."
+    },
+    "groupedBy": {
+      "description": "???",
```
Comment (skoehnen) [inline v3.0/products/fileBundle.schema.json]: Description is missing

Comment (lzehl) [reply v3.0/products/fileBundle.schema.json]: "Concept which was used to group the file instances into this file bundle"

**Comment by lzehl** [comment]
@skoehnen if you add the "description" to the "groupedBy", I can merge this.

File: v3.0/products/fileBundle.schema.json (hunk 1 above)
Comment (lzehl) [reply v3.0/products/fileBundle.schema.json]: @skoehnen if you add the "description" to the "groupedBy", I can merge this.

**Review by lzehl — CHANGES_REQUESTED** [review]

File: v3.0/products/fileBundle.schema.json (hunk 1 above)
Comment (skoehnen) [reply v3.0/products/fileBundle.schema.json]: Done :+1:

**Comment by skoehnen** [comment]
@lzehl Please check if the new location is correct.

**Review by lzehl — APPROVED** [review]

**Pull request merged**